components/layout/block.rs

```diff
@@ -1675,7 +1677,7 @@ impl BlockFlow {
         }
     }
 
-    pub fn has_scrolling_overflow(&self) -> bool {
+    pub fn overflow_will_scroll(&self) -> bool {
```

Comment:
Maybe `style_overflow_may_scroll`, to enhance the difference between this and `has_scrolling_overflow`?